text-input.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Text Input
+sidebar: home_sidebar
+permalink: /text-input/
+---
+
+* TOC Placeholder
+{:toc}
+
+## Introduction
+
+This page describe how to setup basic text input for Flutter applications.
+
+The
+[`Input`](https://docs.flutter.io/flutter/material/Input-class.html)
+widget under
+[Material](https://docs.flutter.io/flutter/material/material-library.html )
+will probably satisfy most text-input use cases. However, `Input` is coupled
+with the Material Design style so there might be cases where you want to use  [`RawInputLine`](https://docs.flutter.io/flutter/widgets/RawInputLine-class.html)
+to achieve desired interfaces outside of the Material Design language.
+
+
+Using the Input Widget
+---------
+
+Assuming that the input is already in focus, the basic data flow for retrieving
+user input will be:
+
+1. User begins to type into the `Input` widget
```

Comment:
I don't think you want "begins" here. What does that mean? Also, do they type into the widget? Or do they use the keyboard?

Perhaps: "User taps a character in the keyboard."